rails/features/teacher_filters_instructional_materials.feature

```diff
@@ -10,11 +10,9 @@ Feature: Teacher filters instructional materials
 
   @javascript @search
   Scenario: Searching Tagged materials
+    # the default data includes grade levels and many tags already
     Given the following Admin::tag records exist:
       | scope         | tag       |
-      | grade_levels  | 5         |
-      | grade_levels  | 7         |
-      | subject_areas | Math      |
```

Comment:
The default data was already creating these 3 tags, which then resulted in duplicates when the test was running.